modules/candidate_parameters/jsx/probandInfo.js

```diff
@@ -103,9 +103,40 @@ var ProbandInfo = React.createClass(
       if (this.state.formData.ProbandGender !== null) {
         dobRequired = true;
       }
-      if (this.state.formData.ProbandDoB !== null) {
-        dob2Required = true;
-      }
+        var dob = this.state.formData.ProbandDoB;
+
+        if (dob !== null && dob !== undefined) {
+            dob2Required = true;
+        }
+
+        var ageDifference = this.state.Data.ageDifference;
+        if (dob !== null && dob !== undefined) {
+
+           var candidateDOB = this.state.Data.candidateDOB;
```

Comment:
These big fancy calculations are very well suited for a dedicated function, preferably with a comment explaining what this actually does.

Then you could simply do something like ```var ageDifference = this.calcAgeDifference()``` in the render method